generated-docs/React/Basic/Events.md

```diff
@@ -35,15 +34,22 @@ input { onChange: handler (preventDefault >>> targetValue)
       }
 ```
 
-_Note: Misusing the `EventFn` *constructor* is UNSAFE and should be avoided -- use the helper functions specific to your platform (such as `React.Basic.DOM.Events`)_
-
 ##### Instances
 ``` purescript
 Semigroupoid EventFn
 Category EventFn
 (IsSymbol l, RowCons l (EventFn a b) fns_rest fns, RowCons l b r_rest r, RowLacks l fns_rest, RowLacks l r_rest, Merge rest fns_rest a r_rest) => Merge (Cons l (EventFn a b) rest) fns a r
 ```
 
+#### `unsafeEventFn`
+
+``` purescript
+unsafeEventFn :: forall a b. (a -> b) -> EventFn a b
+```
+
+Unsafely create an `EventFn`. This function should be avoided.
```

Comment:
Might be worth commenting briefly on why this function is dangerous (it lets SyntheticEvents leave the current scope). Otherwise, this PR looks good to me 👍